library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -50,16 +56,28 @@ sealed class Conversation {
             return when (this) {
                 is V1 -> conversationV1.peerAddress
                 is V2 -> conversationV2.peerAddress
+                is Group -> group.memberAddresses().joinToString(",")
```

Comment:
I'm returning a comma separated list here but I could also throw an error message to use `peerAddresses` instead.

Reply:
I'm OK with throwing an error here. It should only come up when you have enabled the groups flag. Definitely worth calling out in the release notes if we go that way, though.

Reply:
I think I'm just going to leave this as a string for now. Less breaking changes for those not using groups and then when we roll it out I can do more things that break it.